src/ln/channelmanager.rs

```diff
@@ -1413,7 +1417,7 @@ impl ChannelManager {
 						match forward_chan.send_htlc(forward_info.amt_to_forward, forward_info.payment_hash, forward_info.outgoing_cltv_value, htlc_source.clone(), forward_info.onion_packet.unwrap()) {
 							Err(_e) => {
 								let chan_update = self.get_channel_update(forward_chan).unwrap();
-								failed_forwards.push((htlc_source, forward_info.payment_hash, 0x1000 | 7, Some(chan_update)));
+								failed_forwards.push((htlc_source.clone(), forward_info.payment_hash, 0x1000 | 7, Some(chan_update)));
```

Comment:
nit: this line seems stale, compiles fine without the "clone()" due to the continue on the next line.

Reply:
Corrected